packaging/ondemand-bc_access_menu.spec

```diff
@@ -0,0 +1,65 @@
+# Disable debuginfo as it causes issues with bundled gems that build libraries
+%global debug_package %{nil}
+%global repo_name bc_access_menu
+%global app_name1 bc_access_menu_operations
```

Comment:
I would recommend more meaningful variable names.  It would be very easy to accidently use `app_name1` when one might want `app_name2`.  Numbering variables that have meaning can lead to a lot of mistakes.  I'd recommend names like `app_operations` and `app_allocations` so the name is much more tied to purpose.